resources/Messages.resx

```diff
@@ -346,4 +346,40 @@
   <data name="YieldingPackageSource" xml:space="preserve">
     <value>Yielding the package source: '{0}'.</value>
   </data>
+  <data name="FailedToCreateDownloadUri" xml:space="preserve">
+    <value>Failed to make package download URI for package '{0}' and version '{1}'. Failed to download.</value>
+  </data>
+  <data name="JsonSchemaMismatch" xml:space="preserve">
+    <value>The JSON object returned by the NuGet API does not match expected schema. Expected property: '{0}'. Check the debug stream for the full JSON object dump.</value>
+  </data>
+  <data name="JsonObjectDump" xml:space="preserve">
+    <value>JSON object: {0}</value>
+  </data>
+  <data name="FailedToParseTotalHitsCount" xml:space="preserve">
+    <value>Failed to get the number of total hits for query '{0}'. Only returning the first page of responses, if any.</value>
+  </data>
+  <data name="NuGetEndpointDiscovered" xml:space="preserve">
+    <value>Discovered service type '{0}': '{1}' (This will replace the previous URLs of this type)</value>
+  </data>
+  <data name="NuGetEndpointDiscoveredAdditional" xml:space="preserve">
+    <value>Discovered service type '{0}': '{1}' (This will supplement previous URLs of this type as failover)</value>
+  </data>
+  <data name="NuGetEndpointDiscoveryFailed" xml:space="preserve">
+    <value>Failed to discover NuGet endpoints.</value>
+  </data>
+  <data name="TotalPackagesDiscovered" xml:space="preserve">
+    <value>NuGet query returned '{0}' hits.</value>
+  </data>
+  <data name="NuGetServerReadStarted" xml:space="preserve">
+    <value>Reading results from server...</value>
+  </data>
+  <data name="NuGetServerReadProgress" xml:space="preserve">
+    <value>Downloaded '{0} / {1}' results...</value>
+  </data>
+  <data name="VersionIndexDownloadFailed" xml:space="preserve">
+    <value>Failed to download package '{0}' version index to determine list of versions.</value>
+  </data>
+  <data name="AllVersionsSearchNotSupported" xml:space="preserve">
+    <value>When searching using one or more wildcards, the AllVersions switch is not supported. This policy is in place to help alleviate stress on search services.</value>
+  </data>
```

Comment:
Consider getting these messages reviewed by Keith.

Reply:
Sent mail

---
In reply to: [154537933](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154537933) [](ancestors = 154537933)